description: install binaries to /usr/games instead of /usr/bin.
author: Michael Gilbert <mgilbert@debian.org>

--- a/steam.desktop	2014-01-26 20:59:02.663523245 +0000
+++ b/steam.desktop	2014-01-26 21:09:15.447521738 +0000
@@ -1,13 +1,14 @@
 [Desktop Entry]
 Name=Steam
 Comment=Application for managing and playing games on Steam
-Exec=/usr/bin/steam %U
+Exec=/usr/games/steam %U
 Icon=steam
 Terminal=false
 Type=Application
 Categories=Network;FileTransfer;Game;
 MimeType=x-scheme-handler/steam;
 Actions=Store;Community;Library;Servers;Screenshots;News;Settings;BigPicture;Friends;
+Keywords=Games
 
 [Desktop Action Store]
 Name=Store
